kernelci/cli/user.py

```diff
@@ -11,6 +11,23 @@
 from .base_api import APICommand
 
 
+class UserAttributesCommand(APICommand):
```

Comment:
Is this different to how nodes are being searched using arbitrary attributes?  Maybe something more generic for any kind of search (find) query could be used instead.

Reply:
`pylint` was also complaining about it. Added a generic class in `base_api.py`. Bdw I am thinking of a more suitable name for the class than `AttributesCommand`.